rllib/agents/a3c/a3c.py

```diff
@@ -54,6 +54,10 @@ def get_policy_class(config):
 def validate_config(config):
     if config["entropy_coeff"] < 0:
         raise DeprecationWarning("entropy_coeff must be >= 0")
+    if config["sample_async"] and config["framework"] == "torch":
```

Comment:
Had to reintroduce this. It was causing non-determinitic test failures (shape errors).